docs/CHANGELOG.asciidoc

```diff
@@ -206,6 +206,27 @@ Refer to https://www.elastic.co/blog/discover-uses-fields-api-in-7-12[the blog a
 
 // end::notable-breaking-changes[]
 
+[float]
+[[known-issues-v7.12.0]]
+=== Known issues
+
+When upgrading to 7.12.0, Kibana will retry failed saved object migrations without user intervention.
+There are known issues, including users who tried the fleet beta where migrations might fail.
```

Comment:
I'm a little out of my area here, but is this a repeat of line 218? If so, can we delete it?

Reply:
done